test/MileageTest.cpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file MileageTest.cpp
+ * @brief 走行距離の測定用クラスのテスト
+ * @author TanakaShinnosuke
+ */
+
+#include "Mileage.h"
+#include <math.h>
+#include <gtest/gtest.h>
+
+namespace etrobocon2022_test {
+  TEST(MileageTest, calculateMileage)
+  {
+    double radius = 50;
+    double rightAngle = 30;
+    double leftAngle = 40;
+    double rightWheel = 2 * rightAngle * radius * M_PI / 360;
+    double leftWheel = 2 * leftAngle * radius * M_PI / 360;
+    double expected = (rightWheel + leftWheel) / 2;
+    double actual = Mileage::calculateMileage(rightAngle, leftAngle);
+    EXPECT_DOUBLE_EQ(expected, actual);
+  }
+
+  TEST(MileageTest, calculateMileage_minus)
```

Comment:
```suggestion
  TEST(MileageTest, calculateMileageMinus)
```
コーディング規約に書いてなかったと思うんだけど、テスト名にはアンダースコア(_)を使わないようにしてほしい。理由は、宮下さんが勉強会で説明してた通りなんだけど、詳しくは下記参照。
https://qiita.com/hashitaku/items/814fa7641fdec8694d5f